docs/providers/aws/events/streams.md

```diff
@@ -267,6 +267,25 @@ functions:
           parallelizationFactor: 10
 ```
 
+## Setting the FunctionResponseTypes
+
+This configuration allows customers to automatically checkpoint records that have been successfully processed for Amazon Kinesis and Amazon DynamoDB Streams.
+
+For more information, read the [AWS release announcement](https://aws.amazon.com/about-aws/whats-new/2020/12/aws-lambda-launches-checkpointing-for-amazon-kinesis-and-amazon-dynamodb-streams/)
+
+Note: Serverless only sets this property if you explicitly add it to the stream configuration (see example below).
+
+```yml
+functions:
+  preprocess:
+    handler: handler.preprocess
+    events:
+      - stream:
+          arn: arn:aws:dynamodb:region:XXXXXX:table/foo/stream/1970-01-01T00:00:00.000
+          functionResponseTypes:
```

Comment:
I know that underneath it's `FunctionResponseTypes`, but I think for the Framework for now we should go with simple `functionResponseType: ReportBatchItemFailures` setting. I know that AWS might expand that in the future, but I think it's very likely that always at most one `functionResponseType` will be accepted - currently the `FunctionResponseTypes` is limited to always have at most 1 item. 

If AWS ever introduces the option to set multiple types, we will migrate to `functionResponseTypes` and in the meantime, let's use a simpler approach - what do you think?

Btw - we already followed similar approaches for EFS integration for example.